packages/tracing/src/browser/browsertracing.ts

```diff
@@ -209,7 +209,15 @@ export class BrowserTracing implements Integration {
     }
 
     const hub = this._getCurrentHub();
-    const idleTransaction = startIdleTransaction(hub, finalContext, idleTimeout, true);
+    const { location } = getGlobalObject() as WindowOrWorkerGlobalScope & { location: Location };
+
+    const idleTransaction = startIdleTransaction(
```

Comment:
5 arguments 😢

Reply:
You're not wrong. It's one of my goals for v7 to make the entire `IdleTransaction` business a lot simpler.